src/dhtproto/client/UsageExamples.d

```diff
@@ -163,7 +151,7 @@ version ( UnitTest )
                     this.log.error(this.msg_buf);
                     break;
 
-                mixin(handleInvalidCases);
+                mixin(typeof(info).handleInvalidCases);
```

Comment:
I think you should just use `mixin(info.handleInvalidCases)` here.

Reply:
Ah, no sorry, this is fine - it's just the public alias inside the type. However, you've missed one at line 624.

Reply:
(that's why the CI fails).

Reply:
Updated.